### PR TITLE
Task readme and extras

### PR DESCRIPTION
**Pull request by gafda**
Structure, organization and sample readme content generated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,252 @@
...
+
+This section should list any major frameworks/libraries used to bootstrap your project. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+* [![dotNetCore][dotnetcore-shield]][dotnetcore-url]
+* [![Bootstrap][bootstrap-shield]][bootstrap-url]
+* [![Docker][docker-shield]][docker-url]
+* [![Kubernetes][kubernetes-shield]][kubernetes-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (jcoutsousa) [inline README.md:81]: Put here ### Features
List the main features of the project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,252 @@
...
+    ```
+6. Run the project
+    ```sh
+    dotnet run
+    ```
+7. Open your browser and navigate to `http://localhost:5000`
+8. Enjoy!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (jcoutsousa) [inline README.md:156]: ## Structure of the project

How the project is structured.

**Review by jcoutsousa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gafda) [reply README.md:81]: Added.

File: README.md (hunk 2 above)
Comment (gafda) [reply README.md:156]: Do you mean this:
https://github.com/gafda/example-repo/blob/task-readme-and-extras/source/HLD.md

**Review by rubencafe — APPROVED** [review]

**Review by tidoemanuele — APPROVED** [review]

**Pull request merged**